psalm.xml

```diff
@@ -11,11 +11,16 @@
         <ignoreFiles>
             <directory name="vendor" />
             <directory name="src/Stubs" />
+            <directory name="src/cache" />
             <directory name="tests" />
         </ignoreFiles>
     </projectFiles>
 
     <issueHandlers>
         <LessSpecificReturnType errorLevel="info" />
     </issueHandlers>
+    
+    <stubs>
+        <file name="src/Stubs/LumenApplication.stubphp"/>
```

Comment:
I decided to add a stub for the lumen application file, instead  of requiring the lumen framework as a dependency. I assume most people using this package will be using laravel and not lumen, so I think this is acceptable